portfolio/src/main/java/com/google/sps/data/CommentsThread.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class representing the list of comments created on server-dev.html.
+ *
+ * <p>Note: The private variables in this class are converted into JSON.
+ */
+public class CommentsThread {
+
+  /** List of the strings of text in each comment */
+  private final List<String> texts = new ArrayList<>();
+
+  /** List of the image urls each comment (null if no image for a comment) */
+  private final List<String> imageUrls = new ArrayList<>();
+
```

Comment:
Based on my understanding, it's better to have a new data structure like 
class Comment {
  String text;
  String imageUrl;
}

then you have List\<Comment\> in this CommentsThread.

Reply:
I like this a lot more as well. Done.